### PR TITLE
Removed pnpm and tidied up package.json scripts

### PR DESCRIPTION
**Pull request by FunctionDJ**

### CODE REVIEW

**Comment by FunctionDJ** [comment]
I'm requesting changes from
- @igoyak for terraform
- @onetrickwolf for github actions/workflow

What changed
- removed pnpm (now just npm)
- tidied up the [package.json scripts](https://github.com/WarsWorld/WarsWorld/pull/112/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519) to be more straight-forward

File: package.json [hunk 1]
```diff
@@ -5,32 +5,26 @@
   "engines": {
     "node": "^18.15.0"
   },
-  "packageManager": "pnpm@7.27.1",
   "scripts": {
-    "prebuild": "prisma generate && prisma migrate deploy",
-    "build:1-next": "cross-env NODE_ENV=production next build",
-    "build:2-server": "tsc --project tsconfig.server.json",
-    "build": "run-s build:*",
+    "postinstall": "prisma generate",
```
Comment (Pauan) [inline package.json:9]: Doesn't Prisma automatically run `prisma generate` when installing?

**Review by Pauan — APPROVED** [review]
Sad yarn noises.

Eventually we'll need an actual build system, but this is good for now.

**Comment by onetrickwolf** [comment]
@FunctionDJ mades changes but since I am not a maintainer or admin on this project my actions won't run until they are approved and merged. @FemboyDeveloper @FunctionDJ If you guys trust me and want to bump me to maintainer or admin it would probably be faster to debug the actions :)

**Comment by igoyak** [comment]
> I'm requesting changes from
> 
> * @igoyak for terraform
> * @onetrickwolf for github actions/workflow
> 
> What changed
> 
> * removed pnpm (now just npm)
> * tidied up the [package.json scripts](https://github.com/WarsWorld/WarsWorld/pull/112/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519) to be more straight-forward

Just to be clear, your suggesting me to update the terraform instructions/scripts to be compatible with the new build system? I can do that for example after this has been merged.

File: package.json (hunk 1 above)
Comment (FunctionDJ) [reply package.json:9]: i tried without the postinstall and it didn't seem to generate the client automatically.
i'll have to check that out again.

**Comment by FunctionDJ** [comment]
@igoyak oh i thought you could just add commits to this PR. but yeah, that's why i mentioned you. it should be as easy as changing the pnpm part to just using npm and changing a few build scripts. same goes for the github actions.
@onetrickwolf can you add commits to this PR?

**Comment by FunctionDJ** [comment]
@FemboyDeveloper i was hoping that @igoyak could add updated terraform files before merge.

**Comment by FemboyDeveloper** [comment]
> @FemboyDeveloper i was hoping that @igoyak could add updated terraform files before merge.

@FunctionDJ but he said he would do it after the merge? that made me understand this was ready...

**Comment by megagreg72** [comment]
The new linting scripts will not use prettier to check and/or fix non-code files like `README.md`.
However, it still might be worth simplifying the lint scripts like you did, just so it is easier to understand the lint scripts.

File: package.json (hunk 1 above)
Comment (megagreg72) [reply package.json:9]: I just did a clean install after removing this, it worked fine without it 👍

Comment (megagreg72) [reply package.json:9]: hmm, let me doublecheck

Comment (megagreg72) [reply package.json:9]: yup, looks like we dont need it.  I was just making sure that this error appears with or without `prisma generate`
```
[next] [next-auth][error][CLIENT_FETCH_ERROR] 

[next] https://next-auth.js.org/errors#client_fetch_error undefined {

[next]   error: {},

[next]   url: 'http://localhost:3000/api/auth/session',

[next]   message: undefined

[next] }

[next] createContext for unknown user
```
note: I deleted `node_modules` and my docker containers before each test, for a clean build.

Comment (megagreg72) [reply package.json:9]: I'll make a PR to remove it

Comment (megagreg72) [reply package.json:9]: https://github.com/WarsWorld/WarsWorld/pull/119

**Comment by igoyak** [comment]
> > @FemboyDeveloper i was hoping that @igoyak could add updated terraform files before merge.
> 
> @FunctionDJ but he said he would do it after the merge? that made me understand this was ready...

It's not like anything depends on the deployment scripts now, so hopefully fine to do it this way. 
Here we go: https://github.com/WarsWorld/WarsWorld/pull/124

**Pull request merged**